jvm-execution/src/test/scala/com/avast/server/toolkit/execution/ExecutorModuleTest.scala

```diff
@@ -0,0 +1,13 @@
+package com.avast.server.toolkit.execution
+
+import cats.effect.SyncIO
+import org.scalatest.FunSuite
+
+class ExecutorModuleTest extends FunSuite {
+
+  test("ExecutorModule") {
```

Comment:
please provide better description, I don't know what is supposed to be tested here